### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by ispringle**
Restructuring of the project to better align with Python [norms](https://docs.python-guide.org/writing/structure/).

- Moved all `*.py` files under `./bouncer/` and added `__init__.py` because that's how Python rolls
- Added a `requirements.txt`, as this requires on some not-so-standard packages
- Updated `README.md` with installation instructions and updated the usage to follow the new file locations

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pyenchant
```
Comment (drewpearce) [inline requirements.txt:1]: You should target specific versions

File: .vim/coc-settings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (drewpearce) [inline .vim/coc-settings.json]: I don't think it's normative to include IDE settings in the repo.

Comment (plusuncold) [reply .vim/coc-settings.json]: Vim get thee far from me!

File: requirements.txt (hunk 1 above)
Comment (plusuncold) [reply requirements.txt:1]: Noted for the todo list

**Pull request merged**